lib/jsonapi/serializer.ex

```diff
@@ -47,7 +45,7 @@ defmodule JSONAPI.Serializer do
       relationships: %{}
     }
 
-    doc = merge_links(encoded_data, data, view, conn, Application.get_env(:jsonapi, :remove_links, false))
+    doc = merge_links(encoded_data, data, view, conn, remove_links?)
```

Comment:
Could you please update calls to `remove_links?` to `remove_links?()`

Reply:
Fixed!